test/client/before-test.js

```diff
@@ -199,6 +199,24 @@
     window.noop = function () {
     };
 
+    window.checkStringRepresentation = function (wrappedFn, originalFn) {
+        strictEqual(
+            wrappedFn.toString(),
+            originalFn.toString(),
+            'the outputs of the "toString()" method should be the same'
+        );
+        strictEqual(
+            Function.prototype.toString.call(wrappedFn),
```

Comment:
`Function.prototype.toString` and `nativeMethods.Function.prototype.toString` are the same functions

Reply:
Good catch! I found the same problem with `Worker.prototype`, `XMLHttpRequest.prototype` and `Function.prototype` ctor's.